src/renderer/components/+landing-page/landing-page.tsx

```diff
@@ -20,14 +23,17 @@ export class LandingPage extends React.Component {
   }
 
   componentDidMount() {
-    if (!workspaceStore.hasBeenSeen(this.workspace) && this.workspaceClusters.length === 0) {
+    const workspacesSeen = new Set(LandingPage.storage.get());
+
+    if (!workspacesSeen.has(this.workspace.id) && this.workspaceClusters.length === 0) {
       Notifications.info(<><b>Welcome!</b><p>Get started by associating one or more clusters to Lens</p></>, {
         timeout: 30_000,
         id: "landing-welcome"
       });
     }
 
-    workspaceStore.markSeen(this.workspace);
+    workspacesSeen.add(this.workspace.id);
+    LandingPage.storage.set(Array.from(workspacesSeen));
```

Comment:
I think the storage will also need to be updated wherever workspaces are deleted, so they can be removed from this storage.